lib/README.md

```diff
@@ -51,11 +51,6 @@ rely on a file compiled in this directory.
 Contains code for extracting kernel data - i.e. all input and output
 parameters of a kernel invocation.
 
-### [``nan_test``](./nan_test) directory
```

Comment:
Oops! Please could you replace this section with one on `value_range_check` instead.